### PR TITLE
Doc Update

### PR DESCRIPTION
**Pull request by K4orta**
- Document the getHost and getMasheryHost utility functions
- Fix incorrect documentation reference to Github

### CODE REVIEW

File: lib/strategy.js [hunk 1]
```diff
@@ -6,6 +6,19 @@ var OAuth2Strategy = require('passport-oauth2');
...
+/**
+ * `getHost` utility function
+ *
+ * Takes a two character region code and returns a Bnet host URL
+ *
+ *Examples:
+ *
+ *     getHost('eu');  // returns 'eu.battle.net'
+ *
+ * @param {String} region
```
Comment (blizz-aurifex) [inline lib/strategy.js]: Could you add some docs to the region param and return values?

```
@param {String} region Lowercase, two-character representation of the Battle.net region. A list of regions can be found at https://dev.battle.net/docs/read/concepts/Regionality.
@return {String} URL for the specified region's API host.
```

File: lib/strategy.js [hunk 2]
```diff
@@ -6,6 +6,19 @@ var OAuth2Strategy = require('passport-oauth2');
...
+ * @return {String}
+ * @api protected
```
Comment (blizz-aurifex) [inline lib/strategy.js]: This method is not protected.

File: lib/strategy.js [hunk 3]
```diff
@@ -14,6 +27,19 @@ function getHost(region) {
...
+ *
+ * Takes a two character region code and returns a Mashery host URL
+ *
+ *Examples:
+ *
+ *     getMasheryHost('us');  // returns 'us.api.battle.net'
+ *
+ * @param {String} region
+ * @return {String}
+ * @api protected
```
Comment (blizz-aurifex) [inline lib/strategy.js]: This method is not protected.

File: lib/strategy.js [hunk 4]
```diff
@@ -14,6 +27,19 @@ function getHost(region) {
...
+ * `getMasheryHost` utility function
+ *
+ * Takes a two character region code and returns a Mashery host URL
+ *
+ *Examples:
+ *
+ *     getMasheryHost('us');  // returns 'us.api.battle.net'
+ *
+ * @param {String} region
+ * @return {String}
```
Comment (blizz-aurifex) [inline lib/strategy.js]: Could you add some docs to the region param and return values?

```
@param {String} region Lowercase, two-character representation of the Battle.net region. A list of regions can be found at https://dev.battle.net/docs/read/concepts/Regionality.
@return {String} URL for the specified region's Mashery host.
```

File: lib/strategy.js (hunk 3 above)
Comment (K4orta) [inline lib/strategy.js]: What would be a better label for this? `getMasheryHost` is not exported from this module, so I don't think @public applies.

Comment (blizz-aurifex) [inline lib/strategy.js]: Apologies. You're right, that was a mistake on my end. `protected` is accurate.

*sigh* still waking up it seems.

Comment (K4orta) [inline lib/strategy.js]: No prob man, Mondays are hard.

File: lib/strategy.js (hunk 4 above)
Comment (K4orta) [inline lib/strategy.js]: Looks good, I'll add it.

**Pull request merged**